client/scss/components/_icons.scss

```diff
@@ -41,9 +41,14 @@
   }
 }
 
-@each $icon, $content in $icons-after {
-  .icon-#{$icon}:after {
-    content: string.quote(#{$content});
+@each $icon in $icons-after {
+  .icon-#{$icon}-after::after {
+    content: '';
+    mask-image: url('#{$images-root}icons/#{$icon}.svg');
+    width: 1em;
+    height: 1em;
+    display: inline-block;
+    background-color: currentColor;
```

Comment:
This results in the icons looking more or less the same as before. I chose to keep those three "-after" icons done via CSS for two reasons:

- So this keeps working for people implementing listings by copy-pasting Wagtail’s own listings.
- So we introduce appropriate listing components in https://github.com/wagtail/roadmap/issues/33